consumers/src/main/java/uk/gov/moj/sdt/cmc/consumers/CMCConsumerGateway.java

```diff
@@ -37,11 +44,23 @@ public void individualRequest(IIndividualRequest individualRequest,
                                   long connectionTimeOut,
                                   long receiveTimeOut) throws OutageException, TimeoutException {
         LOGGER.debug("Invoke cmc target application service for individual request");
+        String requestType = individualRequest.getRequestType();
+        String sdtRequestReference = individualRequest.getSdtRequestReference();
+        String idamId = ""; // Todo get it from SDTContext
         try {
-            BreathingSpaceRequest request = xmlToObject.convertXmlToObject(individualRequest.getRequestPayload(),
-                                                                           BreathingSpaceRequest.class);
-            BreathingSpaceResponse response = breathingSpace.breathingSpace(request);
-            individualRequest.setRequestStatus(response.getProcessingStatus().name());
+            if(RequestType.BREATHING_SPACE.getType().equals(requestType)) {
+                BreathingSpaceRequest request = xmlToObject.convertXmlToObject(
+                    individualRequest.getRequestPayload(),
+                    BreathingSpaceRequest.class);
+                BreathingSpaceResponse response = breathingSpace.breathingSpace(request);
+                individualRequest.setRequestStatus(response.getProcessingStatus().name());
+            } else if (RequestType.CLAIM_STATUS_UPDATE.getType().equals(requestType)) {
```

Comment:
Just a note that the order of the request types in the if-elseif statement differs between this pull request and the judgment flow pull request (see #113).  This may make the merge tricky.

Reply:
that's fine, we can handle this during the merge